rgw/v2/tests/s3_swift/user_create.py

```diff
@@ -91,6 +98,23 @@ def test_exec(config, ssh_con):
                 if "*** Caught signal (Aborted) **" in out:
                     raise AssertionError("user modify with placementid caused crash!!")
 
+        if config.test_ops.get("user_with_default_placement_and_storageclass", False):
+            (
+                placement_id,
+                storage_class_list,
+            ) = reusable.get_placement_and_storageclass_from_cluster()
+            user_name = "Testuser_defaultstorage"
+            storage_class = config.test_ops.get("storage_class", storage_class_list[0])
+            placement_id = config.test_ops.get("placement_id", placement_id)
+            cmd = f"radosgw-admin user create --uid={user_name} --display-name {user_name} --storage-class {storage_class} --placement-id {placement_id}"
+            out = utils.exec_shell_cmd(cmd)
+            reusable.validate_default_placement_and_storageclass_for_user(
+                user_name, placement_id, storage_class
+            )
+            utils.exec_shell_cmd(
+                f"radosgw-admin user rm --purge-keys --purge-data --uid={user_name}"
```

Comment:
@ckulal ,any specific reason to remove the user? would we not include this suite in multisite?

Reply:
This usecase is not site specific, with multisite this will not cover any additional coverage. hence single site coverage would be sufficient.